src/content/learn/describing-the-ui.md

```diff
@@ -458,28 +458,28 @@ h2 { font-size: 20px; }
 
 <LearnMore path="/learn/rendering-lists">
 
-Read **[Rendering Lists](/learn/rendering-lists)** to learn how to render a list of components, and how to choose a key.
+Baca **[Merender List](/learn/rendering-lists)** untuk mempelajari cara me-*render* daftar komponen dan cara memilih `key`.
 
 </LearnMore>
 
-## Keeping components pure {/*keeping-components-pure*/}
+## Menjaga Komponen Murni {/*keeping-components-pure*/}
 
-Some JavaScript functions are *pure.* A pure function:
+Beberapa fungsi JavaScript adalah murni (*pure*). Sebuah fungsi murni:
 
-* **Minds its own business.** It does not change any objects or variables that existed before it was called.
-* **Same inputs, same output.** Given the same inputs, a pure function should always return the same result.
+* **Memperhatikan urusannya sendiri.** fungsi tidak mengubah objek atau variabel apa pun yang ada sebelumnya dipanggil.
+* **Input sama, output sama.**  Dengan input yang sama, sebuah fungsi murni harus selalu mengembalikan hasil yang sama.
```

Comment:
```suggestion
* **_Input_** sama, _output_ sama.**  Dengan *input* yang sama, sebuah fungsi murni harus selalu memiliki *output* yang sama.
```

Reply:
On another hand, `Input` is a valid Bahasa word. What do you think?

Reply:
It is in our Glossary.
Please refer to that to maintain consistency.

Reply:
I see, should I also change the translation too for this one? @mazipan